--- init.orig	2020-07-11 21:30:14.466736025 +0800
+++ init	2020-07-12 09:19:16.605664952 +0800
@@ -660,10 +660,10 @@
 		CHROOT=${NEW_ROOT}
 	fi
 
-	if [ "${got_good_root}" != '1' ] && [ /dev/nfs != "${REAL_ROOT}" ] && [ sgimips != "${LOOPTYPE}" ] && [ 1 != "${aufs}" ] && [ 1 != "${overlayfs}" ]
-	then
-		bootstrapCD
-	fi
+	# if [ "${got_good_root}" != '1' ] && [ /dev/nfs != "${REAL_ROOT}" ] && [ sgimips != "${LOOPTYPE}" ] && [ 1 != "${aufs}" ] && [ 1 != "${overlayfs}" ]
+	# then
+	# 	bootstrapCD
+	# fi
 
 	if [ "${REAL_ROOT}" = '' ] && [ "${got_good_root}" != '1' ]
 	then
@@ -982,7 +982,7 @@
 	[ -z "${LOOP}" ] && find_loop
 	[ -z "${LOOPTYPE}" ] && find_looptype
 
-	cache_cd_contents
+	# cache_cd_contents
 
 	# If encrypted, find key and mount, otherwise mount as usual
 	if [ -n "${CRYPT_ROOT}" ]
